package.json

```diff
@@ -17,6 +18,7 @@
   "dependencies": {
     "bittorrent-dht": "^10.0.0",
     "bittorrent-tracker": "^9.0.0",
+    "bittorrent-lsd": "^1.0.0",
```

Comment:
This package looks really good! Nice work!!

Reply:
Thank you! 😊